apps/frontend/app/utils/Connections/messageHistoryService.ts

```diff
@@ -0,0 +1,31 @@
+import axios from "axios";
+
+const API_BASE_URL = "http://localhost:4444/messageHistory";
+
+export class MessageHistoryService {
+  // Obtener los mensajes de una comunidad
```

Comment:
If there are going to be comments in the code, please make them in English and on a new line.